app/controllers/application_controller.rb

```diff
@@ -231,7 +231,7 @@ def ajax_request
   def remote_user_provided?
     return false unless Setting["authorize_login_delegation"]
     return false if api_request? && !(Setting["authorize_login_delegation_api"])
-    (@remote_user = request.env["REMOTE_USER"]).present?
+    (@remote_user = request.env["HTTP_REMOTE_USER"]).present?
```

Comment:
Where is `@remote_user` even used? I don't see it anywhere. It looks like this was used until 4e7ea9b86ea7486a2514012ba7101142cc6313de but is now delegated to the abstract SSO provider. Feels like this part should also be delegated to the SSO provider.